contracts/src/v0.8/vrf/dev/testhelpers/VRFV2PlusLoadTestWithMetrics.sol

```diff
@@ -161,4 +165,18 @@ contract VRFV2PlusLoadTestWithMetrics is VRFConsumerBaseV2Plus {
 
     return (_slowestResponseTime, _fastestResponseTime, averageInMillions);
   }
+  
+  function getRequestBlockTimes(uint256 offset, uint256 quantity) external view returns (uint32[] memory) {
```

Comment:
do we need offset and quantity? can we just return all results and when we do `reset()`  we also cleanup this array?

Reply:
This allows us to grab a large chunk at a time, in case the array is very large.

Reply:
but if I want to grab everything that there is ? If I will grab chunk of data, then there is not much value, as it wont represent percentiles fully

Reply:
Right, but nodes have limits to how much data they can return at once so this allows us to work within those limits and still grab as much as we want at any given time.